### ISSUE TITLE
When screen is rotated boinc logo takes most of the space on screen

### ISSUE THREAD

**Issue by okan35** [BOINC/boinc#3320]
As the title says, when you rotate screen on projects list, you see projects through a really small space which is even hard to click and choose because boinc logo takes the whole space on screen.

**Comment by okan35** [comment]
if it is ok, I will just make boinc logo smaller so we can see projects.

**Comment by AenBleidd** [comment]
Yes, please, make the logo smaller on Projects screen when phone is in landscape mode.
